src/swap/utils/getSwapMessage.ts

```diff
@@ -15,18 +15,20 @@ export enum SwapMessage {
 }
 
 export function getSwapMessage({
+  address,
   error,
   from,
   loading,
+  isMissingRequiredFields,
```

Comment:
Can `isMissingRequiredFields` be a life cycle status?

Like do we consider this an important status we should elevate and track across components?

Overall I want to challenge the team to think all our PRs around Life Cycle Status first mindset, and avoid prop hell.

cc @abcrane123 @0xAlec @alessey @cpcramer